recipes-domu/domu-image-fusion/domu-image-fusion.bbappend

```diff
@@ -47,11 +47,7 @@ configure_versions() {
     # corresponding recipies.
     base_update_conf_value ${local_conf} BB_DANGLINGAPPENDS_WARNONLY "yes"
 
-    # hvc0 is not a serial console, so is not processes properly by a modern
-    # start_getty script which is installed for sysvinit based systems.
-    # Instead a distro feature xen should be enabled in a configuration, so a
-    # direct call to getty with hvc0 is installed into inittab by meta-viltualization.
-    base_update_conf_value ${local_conf} SERIAL_CONSOLE ""
+    base_update_conf_value ${local_conf} SERIAL_CONSOLES = "115200;hvc0"
```

Comment:
I believe there are at least two changes:
1. we switch to a newer Yocto and that may require SERIAL_CONSOLES instead of SERIAL_CONSOLE
2. systemd dependency mentioned in the commit message is not honored here: base_update_conf_value will be used for SysV and systemd unconditionally